types/elastic__maki/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for @elastic/maki 6.3
+// Project: https://github.com/elastic/maki
+// Definitions by: Piotr Błażejewicz <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/elastic__maki/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me